### PR TITLE
PBO delegation pool new interface

### PR DESCRIPTION
**Pull request by ghost**
This PR create a new public entry point `initialize_delegation_pool_no_coin` that no coin is supplied in the argument but withdraw from the owner.

### CODE REVIEW

File: aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move [hunk 1]
```diff
@@ -742,6 +745,41 @@ module supra_framework::pbo_delegation_pool {
         staking_config::get_recurring_lockup_duration(&config) / 4
     }
 
+    /// Initialize a delegation pool without actual coin but withdraw from the owner's account.
+    public fun initialize_delegation_pool_no_coin(
```
Comment (ghost) [inline aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move]: with_amout

Comment (sjoshisupra) [reply aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move]: Why has the name not been changed yet? Change it to `_with_amount` @axiongsupra

File: aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move [hunk 2]
```diff
@@ -123,7 +123,7 @@ module supra_framework::pbo_delegation_pool {
     use aptos_std::smart_table::{Self, SmartTable};
     use aptos_std::fixed_point64::{Self, FixedPoint64};
 
-    use supra_framework::coin::Coin;
+    use supra_framework::coin::{Coin, balance};
```
Comment (sjoshisupra) [inline aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move]: Avoid direct import of `balance`, I believe you will still be able to call `coin::balance`, if required you can use `coin::{Self,Coin}`

**Review by sjoshisupra — CHANGES_REQUESTED** [review]

File: aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move (hunk 1 above)
Comment (ghost) [reply aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move]: Sorry, forget to push my local branch to remote.

File: aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move (hunk 2 above)
Comment (ghost) [reply aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move]: Okay, updated.

**Review by sjoshisupra — APPROVED** [review]

**Comment by ghost** [comment]
Looks like the CI keeps time out for new commits today. Not sure why.

File: aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move [hunk 3]
```diff
@@ -742,6 +742,41 @@ module supra_framework::pbo_delegation_pool {
         staking_config::get_recurring_lockup_duration(&config) / 4
     }
 
+    /// Initialize a delegation pool without actual coin but withdraw from the owner's account.
+    public fun initialize_delegation_pool_with_amount(
```
Comment (ghost) [inline aptos-move/framework/supra-framework/sources/pbo_delegation_pool.move:746]: make it entry function.

**Pull request merged**